src/app.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable indent */
```

Comment:
Instead of commenting the `eslint rule` try to correct that issue

Reply:
Done